scio-test/src/test/scala/com/spotify/scio/coders/KryoAtomicCoderTest.scala

```diff
@@ -60,6 +60,10 @@ class KryoAtomicCoderTest extends PipelineSpec {
     cf should roundTrip (Map("a" -> 1, "b" -> 2, "c" -> 3))
   }
 
+  it should "registered case classes" in {
```

Comment:
`registered case classes` doesn't make much sense.

Reply:
Also how are you validating that this is actually using your serializer? Should that be another test?